src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -12,15 +12,25 @@
 using CURLoption = Interop.libcurl.CURLoption;
 using CURLcode = Interop.libcurl.CURLcode;
 using CURLINFO = Interop.libcurl.CURLINFO;
+using CURLProxyType = Interop.libcurl.curl_proxytype;
 
 namespace System.Net.Http
 {
     internal class CurlHandler : HttpMessageHandler
     {
+        #region inner types
```

Comment:
```RequestCompletionSource``` is a nested type but isn't in the same section?  I'd suggest moving this enum down to the end of the file where ```RequestCompletionSource``` is.